lib/src/builder/util.dart

```diff
@@ -42,6 +42,12 @@ Uri idToPackageUri(AssetId id) {
       path: p.url.join(id.package, id.path.replaceFirst('lib/', '')));
 }
 
+Iterable<PartDirective> getNonGeneratedParts(CompilationUnit libraryUnit) {
```

Comment:
Oh yeah one thing I forgot to mention; with this code being added and share, anyone deving on the plugin until we release these changes needs to add a local override to over_react in the plugin directory, otherwise the plugin complains that the method doesn't exist since it's looking in the hosted version.

I wonder if we should not share this logic for now (potentially copying it over and adding a TODO) so that we don't have to worry about it